schema/core.md

```diff
@@ -41,13 +42,19 @@ contain. A simple example is shown below:
 
 ``` json
 {
-  ...
   "name": "IGUIDE Sample Dataset",
   "description": "This is a sample datasets used in the I-GUIDE Catalog documentation",
   "url": "https://my-unique-url.com/9d413b9d1"
 }
 ```
 
+References:
+
```

Comment:
Consider adding these links to the `Schema:name`, `Schema:description`, and `Schema:url` to the text above. For example:

> [Name](https://schema.org/name), [Description](https://schema.org/description), and [URL](https://schema.org/url) are properties inherited from ...

Reply:
Commit 4a218b3 addresses this.